sci-electronics/minipro/minipro-0.7.recipe

```diff
@@ -48,13 +48,13 @@ BUILD_PREREQUIRES="
 
 BUILD()
 {
-	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir make $jobArgs
-	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir make library $jobArgs
+	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir/minipro make $jobArgs
+	DEST_DIR=$installDestDir PREFIX=$prefix SHARE_DIR=$relativeDataDir/minipro make library $jobArgs
 }
 
 INSTALL()
 {
-	DEST_DIR=$installDestDir PREFIX=$prefix BIN_DIR=$relativeBinDir SHARE_DIR=$relativeDataDir MAN_DIR=$relativeManDir make install
+	DEST_DIR=$installDestDir PREFIX=$prefix BIN_DIR=$relativeBinDir SHARE_DIR=$relativeDataDir/minipro MAN_DIR=$relativeManDir/man1 COMPLETIONS_DIR=$prefix/$relativeDataDir/bash-completion/completions make install
```

Comment:
* Same here about DEST_DIR.
 * These lines could really use some line-breaks, they are way over 100 characters.
 * `$prefix/$relativeDataDir` can be replaced with `$dataDir`.

Reply:
I do agree that it looks better with some formatting. I also overlooked the dataDir when looking at the wiki.